src/tests/tests/storage.js

```diff
@@ -191,7 +193,26 @@
     );
   });
 
+  QUnit.test("DNT does not return as an action if user has chosen not to", (assert) => {
```

Comment:
This test doesn't seem to test anything: try getting it to fail by removing the "is DNT checking enabled" checks in the code. The test will still pass because DOMAIN is always seen as NO_TRACKING, not DNT.